tests/basic/pool_function_test.go

```diff
@@ -151,422 +170,422 @@ var _ = Describe("{PoolExpandSmoky}", func() {
+//				orignalTotalSize += pstatus.TotalSize
+//			}
+//		}
+//
+//		originalSizeInBytes = poolToResize.TotalSize
+//		targetSizeInBytes = originalSizeInBytes + 100*units.GiB
+//		targetSizeGiB = targetSizeInBytes / units.GiB
+//
+//		log.InfoD("Current Size of the pool %s is %d GiB. Trying to expand to %v GiB with type add-disk",
+//			poolIDToResize, poolToResize.TotalSize/units.GiB, targetSizeGiB)
+//		triggerPoolExpansion(poolIDToResize, targetSizeGiB, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK)
+//
+//		Step("Ensure pool has been expanded to the expected size", func() {
+//			err = waitForOngoingPoolExpansionToComplete(poolIDToResize)
+//			dash.VerifyFatal(err, nil, "Pool expansion does not result in error")
+//			verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
+//		})
+//
+//		stNodes, err := GetStorageNodes()
+//		log.FailOnError(err, "Unable to get the storage nodes")
+//		var verifyNode node.Node
+//		for _, node := range stNodes {
+//			status, _ := IsPxRunningOnNode(&node)
+//			if node.Id != storageNode.Id && status {
+//				verifyNode = node
+//				break
+//			}
+//		}
+//
+//		// get final total size
+//		provisionStatus, err = GetClusterProvisionStatusOnSpecificNode(verifyNode)
+//		var finalTotalSize float64
+//		for _, pstatus := range provisionStatus {
+//			if pstatus.NodeUUID == storageNode.Id {
+//				finalTotalSize += pstatus.TotalSize
+//			}
+//		}
+//		dash.VerifyFatal(finalTotalSize > orignalTotalSize, true, "Pool expansion failed, pool size is not greater than pool size before expansion")
+//
+//	})
+//
+//})
+//
+//var _ = Describe("{PoolExpansionDiskResizeInvalidSize}", func() {
+//
+//	var testrailID = 34542945
+//	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542945
+//
+//	BeforeEach(func() {
+//		StartTorpedoTest("PoolExpansionDiskResizeInvalidSize",
+//			"Initiate pool expansion using invalid expansion size", nil, testrailID)
+//	})
+//
+//	AfterEach(func() {
+//		EndTorpedoTest()
+//	})
+//
+//	stepLog := "select a pool and expand it by 30000000 GiB with resize-disk type"
+//	It(stepLog, func() {
+//		log.InfoD(stepLog)
+//		// pick pool to resize
+//		pools, err := GetAllPoolsPresent()
+//		log.FailOnError(err, "Unable to get the storage Pools")
+//		pooltoPick := pools[0]
+//
+//		resizeErr := Inst().V.ExpandPool(pooltoPick, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, 30000000, true)
+//		dash.VerifyFatal(resizeErr != nil, true, "Verify error occurs with invalid Pool expansion size")
+//
+//		// Verify error on pool expansion failure
+//		var errMatch error
+//		re := regexp.MustCompile(`.*cannot be expanded beyond maximum size.*`)
+//		if !re.MatchString(fmt.Sprintf("%v", resizeErr)) {
+//			errMatch = fmt.Errorf("failed to verify failure using invalid Pool size")
+//		}
+//		dash.VerifyFatal(errMatch, nil, "Pool expand with invalid PoolUUID failed as expected.")
+//	})
+//
+//})
+//
+//var _ = Describe("{PoolExpandResizeWithSameSize}", func() {
```

Comment:
Why are we removing this test?

Reply:
Because it involves disk-add type pool expansion, which will all be rejected for dmthin